scripts/day2/edge-load-images.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+list="edge-release-images.txt"
+source_registry=""
+
+usage () {
+    echo "USAGE: $0 [--image-list edge-release-images.txt] --source-registry registry.suse.com --registry my.registry.com:5000 --images edge-images.tar.gz"
+    echo "  [-i|--images] tar.gz generated by docker save."
+    echo "  [-s|--source-registry] source registry to pull images from in registry:port format e.g. docker.io."
+    echo "  [-l|--image-list path] text file with list of images; one image per line."
+    echo "  [-r|--registry registry:port] target private registry in the registry:port format."
+    echo "  [-h|--help] Usage message"
+}
+
+POSITIONAL=()
+while [[ $# -gt 0 ]]; do
+    key="$1"
+    case $key in
+        -i|--images)
+        images="$2"
+        shift # past argument
+        shift # past value
+        ;;
+        -r|--registry)
+        target_registry="$2"
+        shift # past argument
+        shift # past value
+        ;;
+        -s|--source-registry)
+        if [ -z "${source_registry}" ]; then
+            source_registry=$2
+        else
+            source_registry="${source_registry},$2"
+        fi
+        shift # past argument
+        shift # past value
+        ;;
+        -l|--image-list)
+        list="$2"
+        shift # past argument
+        shift # past value
+        ;;
+        -h|--help)
+        help="true"
+        shift
+        ;;
+        *)
+        usage
+        exit 1
+        ;;
+    esac
+done
+
+if [[ $help ]]; then
+    usage
+    exit 0
+fi
+
+if [[ -z "${images}" ]]; then
+    echo "Missing '-i|--images'"
+    usage
+    exit 1
+fi
+
+if [[ -z "${target_registry}" ]]; then
+    echo "Missing '-r|--registry'"
+    usage
+    exit 1
+fi
+
+if [[ -z "${list}" ]]; then
+    echo "Missing '-l|--image-list'"
+    usage
+    exit 1
+fi
+
+if [ -z "${source_registry}" ]; then
+    echo "Must have at least one '-s|--source-registry' entry"
+    usage
+    exit 1
+fi
+
+
+echo "Loading ${images}"
+docker load --input ${images}
```

Comment:
podman?

Reply:
Switched to using podman in the scripts. Thanks!